cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -83,7 +81,19 @@ data:
     <match reform.**>
       @type record_reformer
       enable_ruby true
-      tag raw.kubernetes.${tag_suffix[4].split('-')[0..-2].join('-')}
+      <record>
+        # Extract local_resource_id from tag for 'k8s_container' monitored
+        # resource. The format is:
+        # 'k8s_container.<namespace_name>.<pod_name>.<container_name>'.
+        "logging.googleapis.com/local_resource_id" ${"k8s_container.#{tag_suffix[4].rpartition('.')[0].split('_')[1]}.#{tag_suffix[4].rpartition('.')[0].split('_')[0]}.#{tag_suffix[4].rpartition('.')[0].split('_')[2].rpartition('-')[0]}"}
+        # Rename the field 'log' to a more generic field 'message'. This way the
+        # fluent-plugin-google-cloud knows to flatten the field as textPayload
+        # instead of jsonPayload after extracting 'time', 'severity' and
+        # 'stream' from the record.
+        message ${record['log']}
+      </record>
+      tag ${if record['stream'] == 'stderr' then 'raw.kubernetes.stderr' else 'raw.kubernetes.stdout' end}
```

Comment:
Let's open a quick discussion in person today to confirm with the logging backend that we shouldn't use the container name as the tag. Container name is consistent across pods so it wouldn't flood the UI dropdown, and it might be a "better" initial user experience. If we can't get an answer in ~30 minutes I'm fine with proceeding as-is.

Reply:
The requirement to not use the container name as the tag initially came from the Logging Backend folks. Their standpoint probably won't change from the technical perspective. If we do decide to change, it's probably driven by UX experience need. Maybe we should talk to PM?

Reply:
On a second thought, even if we want to fulfill the UX need, the container name based dropped down should be reflected in the UI dropdown `Kubernetes Container => Location => Cluster Name => ...` instead of the logname dropdown. We don't have to include it in the logname (backend will have to index the same thing twice).

Reply:
Per discussion offline with @bmoyles0117 , we will not include container names in the tags. However, the tags should be changed to `stderr` and `stdout` instead of `kubernetes.stderr` and `kubernetes.stdout`.

The change is in. PTAL.